data/strings/strings.txt

```diff
@@ -26763,3 +26763,63 @@
     comment = A number of bookmarks and a number of tracks, separated by comma, like: 1 bookmark, 5 tracks
     tags = android
     en = %@, %@
+
+  [app_tip_00]
+    comment = App tip #00
+    tags = android
+    en = Thank you for using our community-built maps!
+    es = ¡Gracias por usar nuestros mapas creados por la comunidad!
+
+  [app_tip_01]
+    comment = App tip #01
+    tags = android
+    en = With your donations and support, we can create the best maps app in the World!
+    es = ¡Con tus donaciones y apoyo, podemos crear la mejor aplicación de mapas del mundo!
+
+  [app_tip_02]
+    comment = App tip #02
+    tags = android
+    en = Do you like our app? Please donate to support the development! Don't like it yet? Please let us know, and we will fix it!
+    es = ¿Te gusta nuestra aplicación? ¡Por favor dona para apoyar su desarrollo! ¿Todavía no te gusta? ¡Dínoslo y lo arreglaremos!
+
+  [app_tip_03]
+    comment = App tip #03
+    tags = android
+    en = If you know a good software developer, you can ask him to implement a feature that you need.
```

Comment:
it's better to make pronouns ungendered, please change to 'you can ask them to implement'

Reply:
Actually, it'd probably also make sense to remove 'good' from 'a good software developer' to encourage beginners to contribute too :)

Reply:
Thanks @RedAuburn for your comments. I've already modified the strings.

Reply:
@RedAuburn OM codebase is not easy to work on. We're welcoming beginners, but unfortunately, the code quality is not good enough for the production and often has many bugs, and reviewing/fixing introduced bugs takes too much of our time that can be used with better efficiency for the project development.